File: models/src/main/java/ar/edu/itba/paw/models/constants/RequestStatus.java
package ar.edu.itba.paw.models.constants;

public enum RequestStatus {
    PENDING(0),
    ACCEPTED(1),
    REJECTED(2),
    CANCELED(3),
    SOLD(4);

    final private int value;

    RequestStatus(int value) {
        this.value = value;
    }

    public int getValue() {
        return this.value;
    }

    public static int maxValue() {
        return SOLD.ordinal();
    }

    public static int amount() {
        return RequestStatus.values().length;
    }
}

File: webapp/src/main/reactjs/src/constants/requestStatus.js
export const requestStatus = {
    PENDING: 0,
    REJECTED: 2,
    CANCELED: 3,
    SOLD: 4
};

export const requestStatusToString = status => {
    switch (status) {
        case requestStatus.PENDING:
            return 'pending';
        case requestStatus.REJECTED:
            return 'rejected';
        case requestStatus.CANCELED:
            return 'canceled';
        case requestStatus.SOLD:
            return 'sold';
    }
};